turbo.json

```diff
@@ -37,11 +40,19 @@
     },
     "@fedimint/guardian-ui#dev": {
       "cache": false,
-      "dependsOn": ["@fedimint/utils#build", "@fedimint/ui#build"]
+      "dependsOn": [
+        "@fedimint/types#dev",
+        "@fedimint/utils#build",
+        "@fedimint/ui#build"
+      ]
     },
     "@fedimint/gateway-ui#dev": {
       "cache": false,
-      "dependsOn": ["@fedimint/utils#build", "@fedimint/ui#build"]
+      "dependsOn": [
+        "@fedimint/types#dev",
```

Comment:
I don't think you can `dependsOn` a persistent job, I think this would have to be `#build` since it needs to have a terminal state to be a dependent.

Reply:
god catch! fixed